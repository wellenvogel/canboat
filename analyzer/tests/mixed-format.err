DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
DEBUG pgn-test [analyzer] plain_or_fast: fast r=0
DEBUG pgn-test [analyzer] plain_or_fast: fast r=0
DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
DEBUG pgn-test [analyzer] plain_or_fast: fast r=0
DEBUG pgn-test [analyzer] plain_or_fast: plain r=0
