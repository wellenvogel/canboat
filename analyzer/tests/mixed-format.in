2024-07-19-23:24:46.780,3,130824,27,255,8,13,00,33,58,da,0b,41,00
2024-07-19-23:24:46.782,3,130824,27,255,8,14,00,00,00,d0,40,00,00
2024-07-19-23:24:46.782,3,130824,27,255,8,15,00,00,1d,21,ed,03,7f
2024-07-19-23:24:46.782,3,130824,27,255,8,16,20,a6,00,9d,20,45,e3
2024-07-19-23:24:46.782,3,130824,27,255,8,17,19,21,56,2b,2d,41,a4
2024-07-19-23:24:46.782,3,130824,27,255,8,18,b1,07,00,2e,21,b2,13
2024-07-19-23:24:46.782,3,127257,60,255,8,00,ff,7f,1b,00,21,05,ff
2024-07-19-23:24:46.790,2,129025,64,255,8,3b,34,98,16,cc,ea,08,b7
2024-07-19-23:24:46.791,5,130310,35,255,8,ff,8b,72,ff,ff,ff,ff,ff
2024-07-19-23:24:46.869,4,129039,28,255,27,12,e9,9b,2c,14,69,eb,08,b7,34,36,98,16,bf,66,87,06,01,00,00,20,ff,ff,00,70,fe,ff
2024-07-19-23:24:46.913,3,130824,27,255,42,7d,99,7e,20,cb,00,7c,20,00,05,32,21,a8,46,33,21,b5,4d,1d,21,f4,03,50,21,56,5e,51,21,95,0a,52,21,6b,f5,6c,21,56,2c,6d,21,56,2b,ff,ff,ff,ff,ff,ff
2024-07-19-23:24:46.929,3,130824,60,255,8,40,14,7d,99,3c,40,b1,ac
2024-07-19-23:24:46.931,3,130824,60,255,8,41,fd,ff,9e,40,b8,d6,00
2024-07-19-23:24:46.931,3,130824,60,255,8,42,00,44,40,42,7f,f3,ff
2024-07-19-23:24:46.938,5,130312,35,255,8,ff,00,00,8b,72,ff,ff,ff
2024-07-19-23:24:46.869,4,129039,28,255,27,12,e9,9b,2c,14,69,eb,08,b7,34,36,98,16,bf,66,87,06,01,00,00,20,ff,ff,00,70,fe,ff
2024-07-19-23:24:46.791,5,130310,35,255,8,ff,8b,72,ff,ff,ff,ff,ff
