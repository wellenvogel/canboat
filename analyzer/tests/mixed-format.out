2024-07-19-23:24:46.782 3  60 255 127257 Attitude:  SID = 0 (bytes = "00"); Yaw = Unknown (bytes = "FF 7F"); Pitch = 0.2 deg (bytes = "1B 00"); Roll = 7.5 deg (bytes = "21 05")
2024-07-19-23:24:46.790 2  64 255 129025 Position, Rapid Update:  Latitude = 37.9073595 (bytes = "3B 34 98 16"); Longitude = -122.4152372 (bytes = "CC EA 08 B7")
2024-07-19-23:24:46.791 5  35 255 130310 Environmental Parameters (obsolete):  SID = Unknown (bytes = "FF"); Water Temperature = 20.08 C (bytes = "8B 72"); Outside Ambient Air Temperature = Unknown (bytes = "FF FF"); Atmospheric Pressure = Unknown (bytes = "FF FF")
2024-07-19-23:24:46.869 4  28 255 129039 AIS Class B Position Report:  Message ID = Standard Class B position report (bytes = "12", bits = "010010"); Repeat Indicator = Initial (bytes = "00", bits = "00"); User ID = "338467817" (bytes = "E9 9B 2C 14"); Longitude = -122.4152215 (bytes = "69 EB 08 B7"); Latitude = 37.9074100 (bytes = "34 36 98 16"); Position Accuracy = High (bytes = "01", bits = "1"); RAIM = in use (bytes = "02", bits = "1"); Time Stamp = 47 (bytes = "BC", bits = "101111"); COG = 198.6 deg (bytes = "66 87"); SOG = 2.62 m/s (bytes = "06 01"); Communication State = 00 00 00 (bytes = "00 00 00", bits = "0000000000000000000"); AIS Transceiver information = Own information not broadcast (bytes = "20", bits = "00100"); Heading = Unknown (bytes = "FF FF"); Unit type = SOTDMA (bytes = "00", bits = "0"); Integrated Display = No (bytes = "00", bits = "0"); DSC = Yes (bytes = "10", bits = "1"); Band = Entire marine band (bytes = "20", bits = "1"); Can handle Msg 22 = Yes (bytes = "40", bits = "1"); AIS mode = Autonomous (bytes = "00", bits = "0"); AIS communication state = SOTDMA (bytes = "00", bits = "0")
2024-07-19-23:24:46.913 3  27 255 130824 B&G: key-value data:  Manufacturer Code = B & G (bytes = "7D 01", bits = "11001111101"); Industry Code = Marine Industry (bytes = "80", bits = "100"); Key 1 = Polar Speed (bytes = "7E 00", bits = "111101111110"); Length 1 = 2 (bytes = "20", bits = "0010"); Value 1 = 2.03 m/s (bytes = "CB 00"); Key 2 = Polar Performance (bytes = "7C 00", bits = "111001111100"); Length 2 = 2 (bytes = "20", bits = "0010"); Value 2 = 128.0 % (bytes = "00 05"); Key 3 = Opposite Tack COG (bytes = "32 01", bits = "100100110010"); Length 3 = 2 (bytes = "20", bits = "0010"); Value 3 = 103.6 deg (bytes = "A8 46"); Key 4 = Opposite Tack Target Heading (bytes = "33 01", bits = "100100110011"); Length 4 = 2 (bytes = "20", bits = "0010"); Value 4 = 114.0 deg (bytes = "B5 4D"); Key 5 = VMG Performance (bytes = "1D 01", bits = "111000011101"); Length 5 = 2 (bytes = "20", bits = "0010"); Value 5 = 101.2 % (bytes = "F4 03"); Key 6 = Average True Wind Direction (bytes = "50 01", bits = "100001010000"); Length 6 = 2 (bytes = "20", bits = "0010"); Value 6 = 138.4 deg (bytes = "56 5E"); Key 7 = Wind Phase (bytes = "51 01", bits = "100001010001"); Length 7 = 2 (bytes = "20", bits = "0010"); Value 7 = 15.5 deg (bytes = "95 0A"); Key 8 = Wind Lift (bytes = "52 01", bits = "100101010010"); Length 8 = 2 (bytes = "20", bits = "0010"); Value 8 = -15.5 deg (bytes = "6B F5"); Key 9 = 364 (bytes = "6C 01", bits = "011001101100"); Length 9 = 2 (bytes = "20", bits = "0010"); Value 9 = 56 2C (bytes = "56 2C"); Key 10 = 365 (bytes = "6D 01", bits = "011001101101"); Length 10 = 2 (bytes = "20", bits = "0010"); Value 10 = 56 2B (bytes = "56 2B")
2024-07-19-23:24:46.931 3  60 255 130824 B&G: key-value data:  Manufacturer Code = B & G (bytes = "7D 01", bits = "11001111101"); Industry Code = Marine Industry (bytes = "80", bits = "100"); Key 1 = Roll Rate (bytes = "3C 00", bits = "111000111100"); Length 1 = 4 (bytes = "40", bits = "0100"); Value 1 = -873.2 deg (bytes = "B1 AC FD FF"); Key 2 = Pitch Angle (bytes = "9E 00", bits = "111110011110"); Length 2 = 4 (bytes = "40", bits = "0100"); Value 2 = 314.9 deg (bytes = "B8 D6 00 00"); Key 3 = 68 (bytes = "44 00", bits = "001001000100"); Length 3 = 4 (bytes = "40", bits = "0100"); Value 3 = 42 7F F3 FF (bytes = "42 7F F3 FF")
2024-07-19-23:24:46.938 5  35 255 130312 Temperature:  SID = Unknown (bytes = "FF"); Instance = 0 (bytes = "00"); Source = Sea Temperature (bytes = "00"); Actual Temperature = 20.08 C (bytes = "8B 72"); Set Temperature = Unknown (bytes = "FF FF")
2024-07-19-23:24:46.869 4  28 255 129039 AIS Class B Position Report:  Message ID = Standard Class B position report (bytes = "12", bits = "010010"); Repeat Indicator = Initial (bytes = "00", bits = "00"); User ID = "338467817" (bytes = "E9 9B 2C 14"); Longitude = -122.4152215 (bytes = "69 EB 08 B7"); Latitude = 37.9074100 (bytes = "34 36 98 16"); Position Accuracy = High (bytes = "01", bits = "1"); RAIM = in use (bytes = "02", bits = "1"); Time Stamp = 47 (bytes = "BC", bits = "101111"); COG = 198.6 deg (bytes = "66 87"); SOG = 2.62 m/s (bytes = "06 01"); Communication State = 00 00 00 (bytes = "00 00 00", bits = "0000000000000000000"); AIS Transceiver information = Own information not broadcast (bytes = "20", bits = "00100"); Heading = Unknown (bytes = "FF FF"); Unit type = SOTDMA (bytes = "00", bits = "0"); Integrated Display = No (bytes = "00", bits = "0"); DSC = Yes (bytes = "10", bits = "1"); Band = Entire marine band (bytes = "20", bits = "1"); Can handle Msg 22 = Yes (bytes = "40", bits = "1"); AIS mode = Autonomous (bytes = "00", bits = "0"); AIS communication state = SOTDMA (bytes = "00", bits = "0")
2024-07-19-23:24:46.791 5  35 255 130310 Environmental Parameters (obsolete):  SID = Unknown (bytes = "FF"); Water Temperature = 20.08 C (bytes = "8B 72"); Outside Ambient Air Temperature = Unknown (bytes = "FF FF"); Atmospheric Pressure = Unknown (bytes = "FF FF")
